docs/release-procedure.md

```diff
@@ -0,0 +1,208 @@
+   finish, and check that the binary artifacts got attached to the release
+   at `https://github.com/kubevirt/kubevirt/releases/tag/${TAG}`
+   and that the containers were correctly tagged and pushed to
+   <https://quay.io/organization/kubevirt/>
+
+5. If release looks correct, click "edit" on the release in the github UI
+   and uncheck the "This is a pre-release" box. This will make the release
+   official
+
+6. Send a friendly announcement email to <kubevirt-dev@googlegroups.com> using
+   the release notes already present on the release's description in github.
+
+## Creating New Patch Releases
+
+Releases on the stable branch only increment the patch level.
+The release itself is only a git signed tag as it's used for minor releases as well.
+
+1. Create the patch release. Note that the branch is automatically detected.
+
+   `hack/release.sh --new-tag ${TAG}`
+
+2. Wait for [Prow job](https://prow.ci.kubevirt.io/?repo=kubevirt%2Fkubevirt) to
+   finish, and check that the binary artifacts got attached to the release
+   at `https://github.com/kubevirt/kubevirt/releases/tag/$TAG`
+   and that the containers were correctly tagged and pushed to
+   <https://quay.io/organization/kubevirt/>
+
+3. If release looks correct, click "edit" on the release in the github UI
+   and uncheck the "This is a pre-release" box. This will make the release
+   official
+
+4. Send a friendly announcement email to <kubevirt-dev@googlegroups.com> using
+   the release notes already present on the release's description in github.
+
+# Merging to Release Branches
+
+For every release, a branch will be created following the pattern `release-x.y`.
+For now, community members can propose pull requests to be included into a
+stable branch.
+Those pull requests should be limited to bug fixes and must not be
+enhancements. More info related to the policy around backporting can be found
+in this document, [docs/release-branch-backporting.md](https://github.com/kubevirt/kubevirt/blob/main/docs/release-branch-backporting.md)
+
+Cherry picking can be used to pick a merge commit from the main branch
+to a stable branch. An example:
+
+```bash
+git checkout release-0.6
+git cherry-pick $THE_MERGE_COMMIT_ID -m 1 -sx
+[release-0.6 acd756040] Merge pull request #1234 from great_person
+ Author: Bob Builder <builder@bob.com>
+ Date: Thu Jun 28 17:50:05 2018 +0300
+ 5 files changed, 55 insertions(+), 22 deletions(-)
+git push $YOUR_REMOTE release-0.6:release-0.6-aParticularFix
+```
+
+After pushing the branch, you'll need to make sure to create a pull request
+against the correct target branch in GitHub (in this case the target branch
+is `release-0.6`).
+
+# Understanding Release Notes
+
+Release notes are automatically generated by our release tool. The notes are
+sourced from the delta of PRs merged since the last official release. The text
+from those PRs are sourced directly from the `release-notes` section in
+each PRs description.
```

Comment:
```suggestion
each PRs description.

## Using GitHub labels for Release Notes

We use GitHub labels to filter release notes, first using a 'kind/' label and
second using a 'sig/' label where applicable.
In some cases, 'kubevirt-bot' applies one or more of these labels automatically,
however they might need to be manually applied by either the author or a reviewer of the PR.

Add a label by including the ```/kind``` and ```/sig``` commands in the ```release-note-labels``` part of the PR description template. For example:
```
/kind enhancement /sig security /sig compute
```

If a PR matches more than one 'kind/' or 'sig/' label, add all that apply. The release note will be included in whichever takes the greater priority.

As per the PR template, if the PR requires additional action from users switching to the new release, include the string "action required" in the release note. The PR will be flagged accordingly.

## The 'kind/' Labels Used for Release Notes

* ```kind/enhancement```: The PR adds (or mostly adds if it is one of several related PRs) a new feature or enhancement to KubeVirt.
* ```kind/api-change```: The PR adds, removes, or otherwise changes an API.
* ```kind/bug```: The PR fixes a bug.
* ```kind/deprecation```: The PR deprecates a feature that will be removed in a subsequent release.

## The 'sig/' Labels Used for Release Notes

* ```sig/security```: The PR is relevant to SIG Security.
* ```sig/compute```: The PR is relevant to SIG Compute.
* ```sig/network```:  The PR is relevant to SIG Networking.
* ```sig/storage```: The PR is relevant to SIG Storage.
* ```sig/code-quality```: The PR is relevant to SIG Code Quality.

## Viewing Release Notes

Release notes are included in the git tag comment for the release, and copied to the [changelog of the KubeVirt website](https://kubevirt.io/tag/changelog).

```
Looks like this got lost in the rebase as it was split into new file. This ^ should return all the missing text.

Reply:
Thanks. Fixed!